packages/manager/modules/cloud-connect/src/overview/components/datacenter-configuration/template.html

```diff
@@ -0,0 +1,75 @@
+<div class="d-flex flex-row border-bottom">
+    <h4
+        class="oui-heading_4 oui-tile__title"
+        data-ng-bind="$ctrl.datacenter.dcName"
+    ></h4>
+</div>
+<!-- Datacenter details -->
+<div class="oui-tile__body">
+    <div class="table-responsive">
```

Comment:
If the `oui-datagrid` component doesn't suit the needs, we should at least use the styles